### DISCUSSION TITLE
authenticator on a pc

### DISCUSSION THREAD

**Discussion by juliuspetero** [Authenticator-Extension/Authenticator#741] [General]
how can i access my authenticator account on a different device

**Comment by bittner** [comment]
Can you elaborate which OS you're referring to?

-  For Linux distros, e.g., there's [GNOME Authenticator](https://gitlab.gnome.org/World/Authenticator) that has compatible backup/restore for interacting with the Authenticator extension.
- In Chrome (and [now also Firefox](https://github.com/Authenticator-Extension/Authenticator/discussions/614)?) your browser will sync your OTP settings and entries when you're authenticated with your Google or Firefox account.
